### PR TITLE
Add Metro configuration instructions for React Native

### PR DESCRIPTION
**Pull request by Orzech99**
### Summary

This PR updates the `README.md` file in the `matter.js-react-native` package to include necessary Metro bundler configuration instructions for React Native projects.

### Changes

- Added a section to `README.md` detailing the required Metro configuration for resolving `matter.js` packages in React Native.

### Why

Without this configuration, users may encounter module resolution errors when integrating `matter.js` with React Native. This update aims to provide clear guidance to prevent such issues.

### Related Issues

- Fixes module resolution error as discussed in the project’s discord discussions.

### Checklist

- [x] Updated documentation
- [x] Verified changes locally

### CODE REVIEW

File: packages/matter.js-react-native/README.md [hunk 1]
```diff
@@ -23,3 +23,25 @@ This package uses the following react-native libraries to provide the needed fun
...
+
+To use \`**matter.js**\` with React Native, you need to modify your Metro configuration to ensure that the packages are resolved correctly. You need to add a special resolver entry and merge it with your existing Metro configuration. Below is an example of how to do this:
+
+```javascript
+/* eslint-env node */
+const { mergeConfig } = require('@react-native/metro-config');
+const { getSentryExpoConfig } = require('@sentry/react-native/metro');
+
+// Your existing Sentry configuration (or any other configuration you have)
+const sentryConfig = getSentryExpoConfig(__dirname);
```
Comment (Apollon77) [inline packages/matter.js-react-native/README.md:37]: Do you also use snetry? If not maybe better to remove the sentry part because not needed in fact

Comment (Orzech99) [reply packages/matter.js-react-native/README.md:37]: I do not use sentry

Comment (Orzech99) [reply packages/matter.js-react-native/README.md:37]: Should I leave template for user's config or just delete the user config part completely?

**Comment by Apollon77** [comment]
I think reducing to the "minimum" is good

**Review by Apollon77 — APPROVED** [review]

**Pull request merged**